### PR TITLE
Fix/container testing

### PR DESCRIPTION
**Pull request by RV-Adrian**
@ezequielc @alex-deaconu @fjvallarino 
Release tag update, class name updates, and disabled e2e again here. Please review, thanks!

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@RV-Adrian I don't think it's necessary to update the release tag for common-style in this repo. 

Also, wondering why the e2e tests are disabled? Are they failing? And if they are you should just let them fail so we can see what the fail points are in the build.

Thanks.

**Comment by RV-Adrian** [comment]
@alex-deaconu 
I was wondering that this morning. This repo (and others) don't use a release tag. It does add to the upkeep, but also adds a layer of control and visibility. What's the main advantage to leaving it out?

re: e2e, I disabled to get successful build and pull into rise-vision-apps (local component testing is not working for me lately). It's now back in place.

**Comment by alex-deaconu** [comment]
@RV-Adrian regarding release tags for common-style, those are only useful for rise-vision-apps and store repos. This particular repo's tag doesn't really matter as it's overwritten by those. Also, using a blank release tag (as it was before) automatically picks the latest version/tag.

Re: build failures, for dependency repos that don't deploy anywhere, like this one, the build can fail and it will still be grabbed by rise-vision-apps or common-header. 

Basically, you can ignore both aspects (release tag and build failures) when working on these types of repos. Of course, we (devs) will need to fix the failing tests before merging.

Thanks.

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good. Thanks for fixing.

Note: The e2e test that's failing is related to the tag components. Those should just be removed from the repo.

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good 👍 . Make sure you `Squash and Merge` and edit the commit message appropriately. Thanks!
